pkg/controllers/monitoring/monitoring-stack/controller.go

```diff
@@ -143,10 +151,44 @@ func (r *reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 			return ctrl.Result{RequeueAfter: 2 * time.Second}, nil
 		}
 		if err != nil {
-			return ctrl.Result{}, err
+			return r.updateConditions(ctx, req, ms)
```

Comment:
It would be nice to mention the name &kind (in the message) of the resource which failed to reconcile. I am wondering if we can rely on the error message or if we will need to explicitly have/get the resource details. I'm afraid the latter seems more likely.....

Reply:
As a first approach, we can assume that the error will provide enough information. If not the case, we can improve later.